Jenkinsfile

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env groovy
 
-buildPlugin(useContainerAgent: true,
-        configurations: [
-                [platform: 'linux', jdk: '11'],
-                [platform: 'windows', jdk: '11']
-        ])
+buildPlugin(useContainerAgent: true, configurations: [
+  [platform: 'linux', jdk: 17],
```

Comment:
```suggestion
  [platform: 'linux', jdk: 17],
  [platform: 'linux', jdk: 11],
```

Reply:
I'd preferred to not remove Linux Java 11 build and test

Reply:
Inconsistent with the archetype, but OK